### DISCUSSION TITLE
unable to ordering search result.

### DISCUSSION THREAD

**Discussion by odoochina-dev** [community/community#11580] [Other Feature Feedback, Questions, & Ideas] [answered]
cs.github.com is awesome! regex etc could be used.
but one problem: the old normal github search of type code can sort by "Best match" or "Recently Updated" or ..., while cs.github.com seems not able to? (acc: https://cs.github.com/about/syntax)

**Comment by look** [comment, marked as answer]
At this time, we only support ordering results by relevance.

Do you have a use case for other types of ordering? Please let us know.

**Reply by odoochina-dev** [reply]
Thanks for your reply.

Yes, I prefer the "Recently Updated". Actually, need order by commits time, but this is lacking even in old normal github search of type code

**Reply by look** [reply]
It sounds like you would like to see search results ordered by the time the file was last changed. Can you provide more details about what you're trying to accomplish by doing that? What's your goal? Thanks!

**Reply by odoochina-dev** [reply]
Yes, exactly, need the mtime :). But sorry I'm authorized to discuss why we need this publicly. 

btw, If you think the weight or usage this functionality is too small (and not trend to implement it), please just consider "feature parity" of the old normal github search of type code, sorting by "Recently Updated". Thanks!

**Reply by look** [reply]
I'm not sure if recently updated would be helpful for your use case because it is the last time the file was indexed, and we rebuild the index for the technology preview frequently, which would reset this.

**Comment by andreynering** [comment]
Hey there,

I also regularly use the sorting in the old search. My use case is to identify repositories that are using one of my open source projects. In my case I search for `Taskfile.yml` or `Taskfile.yaml` files in repositories:

- https://github.com/search?o=desc&q=filename%3ATaskfile.yml&s=indexed&type=Code&utf8=%E2%9C%93
- https://github.com/search?o=desc&q=filename%3ATaskfile.yaml&s=indexed&type=Code&utf8=%E2%9C%93

An an additional feedback, I miss that the number of stars are not being shown in the results page (seems to also be the case in the new infra as well), and maybe even sorting by number os stars (DESC) would be really interesting.

Another examples for other projects (I keep all these links as favorites in my browser):

- https://github.com/search?o=desc&q=%22go-testfixtures%22+filename%3Ago.mod&s=indexed&type=Code
- https://github.com/search?o=desc&q=%22slim-sprig%22+-indirect+filename%3Ago.mod&s=indexed&type=Code

Maybe it's not a common use case or the intention of the new search, but I decided to leave this feedback anyway.
